docs/changes.rst

```diff
@@ -144,6 +144,9 @@ Released: not yet
 * Fix pywem_mock issue with Delete class not calling providers to handle
   the DeleteInstance (see issue #2643)
 
+* Test: Workaround for BadStatusLine issue in test_WBEMListener_send_indications
+  test function. (issue #2659)
```

Comment:
I would clarify by stating that this is NOT a fix for the issue itself, This is just to be sure users understand.

Reply:
DONE